app/models/answer_set.rb

```diff
@@ -2,4 +2,21 @@
 
 # Corresponds with a multilevel questioning. An AnswerSet's parent is an AnswerGroup. Its children are Answers
 class AnswerSet < ResponseNode
+  belongs_to :questioning
+
+  alias answers c
```

Comment:
Alias this to `children` instead of `c`? They both exist and it's a bit more descriptive...